packages/core/src/components/dyte-participants-waiting-list/dyte-participants-waiting-list.tsx

```diff
@@ -105,14 +105,25 @@ export class DyteParticipantsWaitlisted {
 
   private shouldShowWaitlist = () => {
     if (this.meeting.meta.viewType === 'LIVESTREAM') return false;
-    return (
-      this.meeting.self.permissions.acceptWaitingRequests &&
-      this.waitlistedParticipants.length !== 0
-    );
```

Comment:
Same, should we show this if there are no requests?

Reply:
Done. We are not showing it if there are no requests.